src/azure-cli-core/azure/cli/core/tests/test_progress.py

```diff
@@ -60,7 +60,7 @@ def test_progress_indicator_indet_stdview(self):
         before = view.spinner.total
         view.write({})
         after = view.spinner.total
-        self.assertTrue(after >= before)
+        self.assertTrue(after == before)
```

Comment:
In humanfriendly 4.7 `spinner.total` is always the default value 0. In 8.0, the default value is `None` which makes `>=` invalid.